packaging/ubuntu-16.04/changelog

```diff
@@ -1,3 +1,18 @@
+snapd (2.60.1) xenial; urgency=medium
+
+  * New upstream release, LP: #2024007
+    - install: fallback to lazy unmount() in writeFilesystemContent
+    - data: include "modprobe.d" and "modules-load.d" in preseeded blob
+    - gadget: fix install test on armhf
+    - interfaces: fix typo in network_manager_observe
+    - sandbox/apparmor: don't let vendored apparmor conflict with system
+    - gadget/update: set parts in laid out data from the ones matched
+    - many: move SnapConfineAppArmorDir from dirs to sandbox/apparmor
+    - many: stop using `-O no-expr-simplify` in apparmor_parser
```

Comment:
should this mention that is a stability vs speed trade-off atm?

Reply:
Good point, let me add it

Reply:
I updated NEWS.md to include a more human friendly summary of the changes and their impact